release.nix

```diff
@@ -2,7 +2,7 @@
   # Passed in by Hydra depending on the configuration, contains the revision and the out path
 , plutus ? null
 , rootsOnly ? false
-, checkMaterialization ? true
+, checkMaterialization ? false
```

Comment:
Can you change the default in `ci.nix` too, so they're consistent? Also maybe a comment here saying that we *do* check this, but elsewhere in the github action.

Reply:
Sure. What's the use case for ci.nix vs release.nix?

Reply:
Historical reasons. We did a trial run of Hercules CI which uses `ci.nix`. We could probably inline it into `release.nix` these days. Also `ci.nix` has a simpler structure: it's just a nested set of derivations, rather than the whole "jobset" thing for hydra, so it's sort of nice having it as its own thing.